src/pages/instances/actions/InstanceBulkDelete.tsx

```diff
@@ -79,90 +79,53 @@ const InstanceBulkDelete: FC<Props> = ({ instances, onStart, onFinish }) => {
     });
   };
 
-  const getStoppedInstances = () => {
-    if (!deleteCount) {
-      return null;
+  const getBulkDeleteBreakdown = () => {
+    if (!(ignoredCount + restrictedCount)) {
```

Comment:
```suggestion
    if (ignoredCount + restrictedCount === 0) {
```
This seems easier to read.